backend/benefit/applications/services/ahjo_integration.py

```diff
@@ -554,17 +558,25 @@ def send_new_attachment_records_to_ahjo(
     return result, response_text
 
 
-class DecisionProposalError(Exception):
-    """Custom exception for errors in XML generation."""
-
-    pass
-
-
 def send_decision_proposal_to_ahjo(
     application: Application, ahjo_token: AhjoToken
 ) -> Union[Tuple[Application, str], None]:
     """Send a decision proposal and it's XML attachments to Ahjo."""
 
+    # https://helsinkisolutionoffice.atlassian.net/browse/HL-1558
+    # Check if the application already has a decision proposal accepted in Ahjo status,
+    # bail out if it does
+
+    if application.ahjo_status.filter(status=AhjoStatusEnum.DECISION_PROPOSAL_ACCEPTED):
```

Comment:
Sometimes a new record is sent at the same time as a decision proposal is accepted in ahjo, and the latest status is AhjoStatusEnum.NEW_RECORDS_RECEIVED while the second to latest status is AhjoStatusEnum.DECISION_PROPOSAL_ACCEPTED, this check prevents a new decision proposal being sent in that case.